src/sentry/issues/auto_source_code_config/code_mapping.py

```diff
@@ -368,49 +380,37 @@ def convert_stacktrace_frame_path_to_source_path(
 
 
 def create_code_mapping(
-    installation: IntegrationInstallation,
+    organization: Organization,
     project: Project,
-    code_mapping: CodeMapping,
+    stacktrace_root: str,
+    source_path: str,
+    repo_name: str,
+    branch: str,
 ) -> RepositoryProjectPathConfig:
-    organization_integration = installation.org_integration
-    if not organization_integration:
+    installation = get_installation(organization)
+    # It helps with typing since org_integration can be None
+    if not installation.org_integration:
         raise InstallationNotFoundError
 
     repository, _ = Repository.objects.get_or_create(
-        name=code_mapping.repo.name,
-        organization_id=organization_integration.organization_id,
-        defaults={
-            "integration_id": organization_integration.integration_id,
```

Comment:
Using `installation.model.id` instead of this.

https://github.com/getsentry/sentry/blob/84d50b384b70b3cbc867e3bbe41192aaacc56399/src/sentry/integrations/models/organization_integration.py#L30
https://github.com/getsentry/sentry/blob/84d50b384b70b3cbc867e3bbe41192aaacc56399/src/sentry/integrations/base.py#L357